modules/core/src/main/scala/org/scalasteward/core/application/Config.scala

```diff
@@ -156,9 +156,11 @@ object Config {
   final case class GitHubCfg(
   ) extends ForgeSpecificCfg
 
+  final case class MergeRequestApprovalRulesCfg(approvalRuleName: String, requiredApprovals: Int)
+
   final case class GitLabCfg(
       mergeWhenPipelineSucceeds: Boolean,
-      requiredReviewers: Option[Int],
+      requiredApprovals: Option[Either[Int, Nel[MergeRequestApprovalRulesCfg]]],
```

Comment:
Here is how to read this;
Left side the old flow, user just provides the old configuration via `--gitlab-required-reviewers <integer>`
Right side is the new flow, user provides the new configuration via `--merge-request-level-approval-rule <approvals_rule_name=required_approvals>`

I also renamed the name of this variable. It's not reviewers, it's actually approvals. 

I can wrap left side to a value class. It might be more explanatory then just Int. WDYT?

Reply:
@endertunc I'd suggest a `-gitlab` prefix for this new config param perhaps, for consistency?